playbooks/capsule/tasks/main.yaml

```diff
@@ -0,0 +1,65 @@
+---
+- name: 'Set SELinux to Permissive'
+  selinux: policy=targeted state=permissive
+
+- name: 'Get Server Hostname'
+  shell: hostname -f
+  delegate_to: "{{ server }}"
+  register: server_fqdn
+- name: 'Get Server IP Address'
+  shell: ifconfig eth0 2>/dev/null|awk '/inet/ {print $2}'|sed 's/addr://'
```

Comment:
Quick googling found this http://serverfault.com/questions/638507/how-to-access-host-variable-of-a-different-host-with-ansible (it even has the example with the ip address)